src/dbnode/persist/fs/seek_manager.go

```diff
@@ -123,28 +123,59 @@ type seekersAndBloom struct {
 	volume      int
 }
 
+func (s seekersAndBloom) closeWithLock() error {
+	multiErr := xerrors.NewMultiError()
+	for _, seeker := range s.seekers {
+		multiErr = multiErr.Add(seeker.seeker.Close())
+	}
+	return multiErr.FinalError()
+}
+
+// Returns true if any seekers are borrowed.
+func (s seekersAndBloom) anyBorrowed() bool {
```

Comment:
Rename this to anyBorrowedWithLock since this method should be invoked under byTime lock